src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+struct FirstTicketValidationConsentViewModel {
+
+	init(
+		serviceProvider: String,
+		subject: String,
+		onDataPrivacyTap: @escaping () -> Void
+	) {
+		self.serviceProvider = serviceProvider
+		self.subject = subject
+		self.onDataPrivacyTap = onDataPrivacyTap
+	}
+
+	// MARK: - Internal
+
+	var dynamicTableViewModel: DynamicTableViewModel {
+		DynamicTableViewModel([
+			// Data privacy cell
+			.section(
+				separators: .all,
+				cells: [
+					.body(
+						text: AppStrings.ContactDiary.Information.dataPrivacyTitle,
```

Comment:
Perhaps a new AppStrings for ticketValidation or you could reuse the one I already created for ticketValidation...

Reply:
This is the same file I think so you have already changed it in your PR :)

Reply:
and i thought: Ah looks quite familiar 